sale_stock_available_to_promise_release_block/views/sale_order_line.xml

```diff
@@ -42,7 +42,8 @@
     <field name="state">code</field>
     <field name="code">
     if records:
-        records.move_ids.action_unblock_release()
+      for move in records.move_ids:
+        move.action_unblock_release()
```

Comment:
No need to change this, the `action_unblock_release` has no singleton issue, see:
https://github.com/OCA/wms/blob/16.0/stock_available_to_promise_release_block/models/stock_move.py#L42-L44